Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -13,10 +13,15 @@ private enum SearchListSection: Int {
     case Search
 }
 
+protocol SearchViewControllerDelegate: class {
+    func searchViewController(searchViewController: SearchViewController, didSelectURL url: NSURL)
+}
+
 class SearchViewController: SiteTableViewController {
     private var sortedEngines = [OpenSearchEngine]()
     private var suggestClient: SearchSuggestClient?
     private var searchSuggestions = [String]()
+    var delegate: SearchViewControllerDelegate?
```

Comment:
Oops, this was supposed to be in a later patch where `SearchViewController` is changed to extend `UIViewController` instead of `SiteTableViewController` (which is fine since `UIViewController` has no delegate that would be shadowed). So yeah, shouldn't make this change here since shadowing the `SiteTableViewController`'s delegate could be an issue.